README.md

```diff
@@ -84,6 +84,30 @@ To change allowed key exchange algorithms to a specific set, you can add a
 }
 ```
 
+To change allowed MACs to a specific set, you can add a `macs` section:
+
+```json
+{
+  "ssh": {
+    "authorized-keys...",
+    "macs": [
+      "hmac-sha2-256",
+      "hmac-sha2-512",
+      "hmac-md5",
+      "hmac-md5-96",
```

Comment:
maybe drop weak md5 hmacs from the example?

Reply:
Good call out. And you know, rather than give an explicit list example maybe show how one might use the setting to tweak the default? Is that intended to be supported here @mlacko64 ?

ie: if we consider https://man.openbsd.org/sshd_config#MACs

> If the specified list begins with a ‘+’ character, then the specified algorithms will be appended to the default set instead of replacing them. If the specified list begins with a ‘-’ character, then the specified algorithms (including wildcards) will be removed from the default set instead of replacing them. If the specified list begins with a ‘^’ character, then the specified algorithms will be placed at the head of the default set.

I think we'd be able to express surgical "edits" to the defaults - maybe even this would be a model default setting? Something like this:

```json
{
  "ssh": {
    "macs": [
      "-hmac-sha1"
    ]
  }
}
```

This way, any Bottlerocket settings (ie: other than build time defaults of OpenSSH itself) would be able to augment rather than exhaustively set the list. Settings could even be provided in the model to default _remove_ insecure elements if/when the need arises.

In fact, after chatting with @jpculp I think this might apply to a few of the other "list" settings in `ssh`.